mono/mini/interp/interp.c

```diff
@@ -334,6 +334,28 @@ mono_interp_get_imethod (MonoDomain *domain, MonoMethod *method, MonoError *erro
 	return rtm;
 }
 
+static MONO_NEVER_INLINE void
+interp_return_stub ()
+{
+	return;
+}
+
+static MONO_NEVER_INLINE void
+interp_push_lmf_with_ctx (MonoLMFExt *ext, InterpFrame *frame)
+{
+	memset (ext, 0, sizeof (MonoLMFExt));
+	ext->interp_exit = 2;
+	ext->interp_exit_data = frame;
+
+#ifdef MONO_INIT_CONTEXT_FROM_CURRENT
+	MONO_INIT_CONTEXT_FROM_CURRENT (&ext->ctx);
```

Comment:
@lewurm I don't understand the point of this define path

Reply:
it's copy/paste from other places that are using `MONO_INIT_CONTEXT_FROM_FUNC`. we could probably remove everything that mentions `MONO_INIT_CONTEXT_FROM_CURRENT` from the code base, no architecture seems to define it.